### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by kelfro**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: frontend/src/pages/Settings/Settings.tsx [hunk 1]
```diff
@@ -83,6 +83,7 @@ const Settings: React.SFC<RouteComponentProps> = (props) => {
     axios
       .get(`${url}/users`, headers)
       .then(({ data }) => {
+        console.log(data);
```
Comment (NandoTheessen) [inline frontend/src/pages/Settings/Settings.tsx]: Can you remove this please?

File: frontend/src/pages/Landingpage.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import React from 'react';
```
Comment (NandoTheessen) [inline frontend/src/pages/Landingpage.tsx]: This whole file now lives in /src/pages/Landingpage/Landingpage.tsx, please pull from master / if you already have it, delete this file

File: backend/src/app.ts [hunk 3]
```diff
@@ -75,6 +75,11 @@ server
 server.route('/itemComplete').post(items.itemComplete);
```
Comment (NandoTheessen) [inline backend/src/app.ts]: I don't feel these changes should be in this PR, did you merge Ronalds branch by chance?

File: frontend/src/components/Sidebar/Sidebar.tsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ const Sidebar = ({ onClick }: LinkProps) => {
         <StyledUL>
           <li>
             <StyledLink to='/reports'>
-              <h4>Reports</h4>
+              <button><h4>Reports</h4></button>
```
Comment (NandoTheessen) [inline frontend/src/components/Sidebar/Sidebar.tsx]: Please revert this change.

File: frontend/src/components/Sidebar/Sidebar.styling.tsx [hunk 5]
```diff
@@ -38,10 +38,21 @@ const StyledLink = styled(NavLink)`
     }
 `;
 
+const Accordion = styled('button')`
```
Comment (NandoTheessen) [inline frontend/src/components/Sidebar/Sidebar.styling.tsx]: Is there a reason we are styling a button and naming it accordion? 
Why would it be in the Sidebar (headerbar) ?

File: frontend/src/pages/Settings/Settings.tsx (hunk 1 above)
Comment (kelfro) [reply frontend/src/pages/Settings/Settings.tsx]: I rebased the whole thing so all these things should be fixed.
